### PR TITLE
Translations

### PR DESCRIPTION
**Pull request by phattya**
# Pull Request Template

## Issue Overview
Translations need to be added/updated from Localize human translations.

This PR addresses #issue

- [X] This change addresses the issue in full
- [ ] This change addresses only certain aspects of the issue
- [ ] This change is a dependency for another issue
- [ ] This change has a dependency from another issue

## Description
Translations need to be added/updated from Localize human translations.

## How Can This Be Tested/Reviewed?
Just look thru the diff & see if anything jumps out at you.

## Checklist:

- [x] My code follows the style guidelines of this project
- [ ] I have added QA notes to the issue with applicable URLs
- [x] I have performed a self-review of my own code
- [x] I have reviewed the changes in a desktop view
- [x] I have reviewed the changes in a mobile view
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules
- [x] I have assigned reviewers
- [ ] I have run `yarn generate:client` and/or created a migration if I made backend changes that require them
- [x] My commit message(s) is/are polished, and any breaking changes are indicated in the message and are well-described
- [ ] Commits made across packages purposefully have the same commit message/version change, else are separated into different commits

## Reviewer Notes:

Steps to review a PR:

- Read and understand the issue, and ensure the author has added QA notes
- Review the code itself from a style point of view
- Pull the changes down locally and test that the acceptance criteria is met
- Also review the acceptance criteria on the Netlify deploy preview (noting that these do not yet include any backend changes made in the PR)
- Either explicitly ask a clarifying question, request changes, or approve the PR if there are small remaining changes but the PR is otherwise good to go

## On Merge:

If you have one commit and message, squash. If you need each message to be applied, rebase and merge.

### CODE REVIEW

File: sites/partners/src/components/listings/PaperListingForm/sections/BuildingDetails.tsx [hunk 1]
```diff
@@ -144,6 +144,10 @@ const BuildingDetails = ({
   }, [mapPinPosition])
```
Comment (phattya) [inline sites/partners/src/components/listings/PaperListingForm/sections/BuildingDetails.tsx:144]: Need to rebase after PR merged for this,  so this disappears

File: shared-helpers/src/locales/es.json [hunk 2]
```diff
@@ -405,11 +405,6 @@
   "counties.Santa Clara": "Santa Clara",
   "counties.Solano": "Solano",
   "counties.Sonoma": "Sonoma",
-  "county.goToOtherPortalsTitle": "<to add after Localize translation>",
-  "county.goToOtherPortalsHover": "<to add after Localize translation>",
-  "county.goToOtherPortalsAlamedaSanMateo": "<to add after Localize translation>",
-  "county.goToOtherPortalsCitySanJose": "<to add after Localize translation>",
-  "county.goToOtherPortalsSanFrancisco": "<to add after Localize translation>",
   "eligibility.accessibility.acInUnit": "CA en la unidad",
```
Comment (phattya) [inline shared-helpers/src/locales/es.json:408]: Translations not needed for the Partner portal

File: shared-helpers/src/locales/tl.json [hunk 3]
```diff
@@ -357,12 +357,7 @@
   "counties.San Mateo": "San Mateo",
   "counties.Santa Clara": "Santa Clara",
   "counties.Solano": "Solano",
-  "counties.Sonoma": "Sonoma",
-  "county.goToOtherPortalsTitle": "<to add after Localize translation>",
-  "county.goToOtherPortalsHover": "<to add after Localize translation>",
-  "county.goToOtherPortalsAlamedaSanMateo": "<to add after Localize translation>",
-  "county.goToOtherPortalsCitySanJose": "<to add after Localize translation>",
-  "county.goToOtherPortalsSanFrancisco": "<to add after Localize translation>",
+  "counties.Sonoma": "Sonoma",
```
Comment (phattya) [inline shared-helpers/src/locales/tl.json:360]: Translations not needed for the Partner portal

File: shared-helpers/src/locales/zh.json [hunk 4]
```diff
@@ -405,11 +405,6 @@
   "counties.Santa Clara": "Santa Clara (聖克拉拉)",
   "counties.Solano": "Solano (索拉諾)",
   "counties.Sonoma": "Sonoma (索諾瑪)",
-  "county.goToOtherPortalsTitle": "<to add after Localize translation>",
-  "county.goToOtherPortalsHover": "<to add after Localize translation>",
-  "county.goToOtherPortalsAlamedaSanMateo": "<to add after Localize translation>",
-  "county.goToOtherPortalsCitySanJose": "<to add after Localize translation>",
-  "county.goToOtherPortalsSanFrancisco": "<to add after Localize translation>",
```
Comment (phattya) [inline shared-helpers/src/locales/zh.json:412]: Translations not needed for the Partner portal

File: shared-helpers/src/locales/es.json [hunk 5]
```diff
@@ -750,7 +750,7 @@
   "listings.applications": "Solicitudes",
   "listings.apply.applicationWillBeAvailableOn": "La solicitud podrá ser descargada y recogida el %{openDate}",
   "listings.apply.applyOnline": "Haga su solicitud por Internet",
-  "listings.apply.applyOnlineMessage": "<to add after Localize translation>",
```
Comment (ifranch) [inline shared-helpers/src/locales/es.json:753]: listado? Fine with me if we don't want to standardize it though

File: shared-helpers/src/locales/es.json [hunk 6]
```diff
@@ -1018,7 +1019,7 @@
   "t.neighborhood": "Comunidad",
   "t.next": "Siguiente",
   "t.no": "No",
-  "t.noMatchingListings": "No hay listados coincidentes",
+  "t.noMatchingListings": "No hay ningún listados coincidentes",
```
Comment (ifranch) [inline shared-helpers/src/locales/es.json:1022]: This should say "listado coincidente" in singular, current grammar is wrong

**Review by ifranch — APPROVED** [review]

File: shared-helpers/src/locales/es.json (hunk 5 above)
Comment (phattya) [reply shared-helpers/src/locales/es.json:753]: great catch! Fixed. Thanks!

File: shared-helpers/src/locales/es.json (hunk 6 above)
Comment (phattya) [reply shared-helpers/src/locales/es.json]: Fixed! thanks!

**Review by isaac-looker — APPROVED** [review]

**Pull request merged**